robolectric/src/test/java/org/robolectric/internal/bytecode/InstrumentingClassLoaderTest.java

```diff
@@ -441,78 +440,91 @@ public void shouldInterceptFilteredStaticMethodInvocations() throws Exception {
 
   @Test
   public void byte_shouldBeHandledAsReturnValueFromInterceptHandler() throws Exception {
+    if (InvokeDynamic.ENABLED) return;
```

Comment:
I don't think it is a good idea, since the tests should still be run even if a developer/a CI uses the JVM compatible with invokedynamic.